### ISSUE TITLE
Save favicons and logos for news sources

### ISSUE THREAD

**Issue by embray** [hayj/Renewal#2]
Working on overhauling the news feed UI.  There is a field for the name of the news source, but many of them also have favicons and/or logos, either in their metadata, or that can be scraped from their websites.  This would be a nice addition for quick visual recognition of news sources.

**Comment by hayj** [comment]
👍 nice idea

**Comment by embray** [comment]
This is partially implemented now, as of 40c9236 -- the fake data I'm using for article sources includes their logos, which are now displayed alongside the articles.  However, for this to really work will of course require support on the backend to cache logos for all sources, and probably also implementing some caching on the client side as well.

**Comment by embray** [comment]
@hayj By the way, if you get a chance can you add me as a maintainer on this repository?  I would like to be able to create labels and issues and things like that.

**Comment by hayj** [comment]
Yes a caching mechanism would be useful, I added you as maintainer ;)
